addons/web/static/src/js/core/ajax.js

```diff
@@ -352,37 +352,82 @@ function post (controller_url, data) {
     return Def;
 }
 
+/**
+ * Loads an XML file according to the given URL and adds its associated qweb
+ * templates to the given qweb engine. The function can also be used to get
+ * the deferred which indicates when all the calls to the function are finished.
+ *
+ * Note: "all the calls" = the calls that happened before the current no-args
+ * one + the calls that will happen after but when the previous ones are not
+ * finished yet.
+ *
+ * @param {string} [url] - an URL where to find qweb templates
```

Comment:
it is an optional parameter, but it should be explained what happens when no url is given